scripts/utils/constants.ts

```diff
@@ -1,21 +1,11 @@
 /**
  * Addresses
  */
-export const ETH_USDC_POOL = "0x8ad599c3A0ff1De082011EFDDc58f1908eb6e6D8";
-export const MAINNET_ETH_ORACLE = "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419";
-export const KOVAN_ETH_ORACLE = "0x9326BFA02ADD2366b30bacB125260Af641031331";
-export const KOVAN_WETH = "0xd0A1E359811322d97991E03f863a0C30C2cF029C";
```

Comment:
why do we remove these

Reply:
These variables are already defined in `constants/constants.ts` and we should only have 1 source of truth, not define the variables in multiple files.

Ideally, I would like to remove `scripts/utils/constants.ts` in the future and have it all defined in constants/constants.ts`